packages/main/src/ColorPicker.ts

```diff
@@ -142,11 +148,11 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 	_mainValue: ColorRGB;
 
 	/**
-	 * Defines the currenty selected color from the main color section.
+	 * Defines the currenty selected color.
 	 * @private
 	 */
-	@property({ type: Object })
-	_value: ColorRGB = getRGBColor(this.value);
+	@property({ type: Boolean })
```

Comment:
Isn't type `Object` the accurate type?

Reply:
you're right, it probably got messed up when i was merging changes from master, and I didnt catch it